.gitignore

```diff
@@ -1,4 +1,5 @@
 .bundle/
+.vscode/
```

Comment:
Can you add this to the excludes in your local repo instead (or your global `.gitignore` if you'd prefer)? I used to have `.idea` files checked in but came to the conclusion that with editors being personal choices on this kind of project it didn't make much sense.

Reply:
I had no idea there was an excludes file! I'll do that. Didn't want it in global `.gitignore` for other reasons.